### ISSUE TITLE
Fool's Gold Doesn't Have proper Ele STR

### ISSUE THREAD

**Issue by Kagekiki** [EdenServer/community#3580]
### Checklist

- [X ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [X ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ X] I have provided my client version in the details. (type /ver into your game window)

### Details

Hell Guys small thing The fools Gold for GSM +1 doesn't have proper Light Ele Str It is supposed to be 9 but is being overwritten by all other +9 light str furniture (Crystal rose etc. )

**Comment by 59blargedy** [comment]
it is set to 9. Im unfamiliar with moghancements/auras - are you saying that equal valued light auras should defer to a certain moghancement or ? is the issue that your moghancement isnt registering, or what exactly is the issue?

**Comment by Kagekiki** [comment]
Any 9 is overwriting the goldsmith moghancement so the crystal rose and
marble plaques

**Comment by 59blargedy** [comment]
it looks like it is currently working the opposite of wiki, in that i was able to obtain the enhancement when placing fool's gold last. changed the title to reflect.

**Comment by Kagekiki** [comment]
Yeah when I changed them didn't seem to help ill try again.

**Comment by Kagekiki** [comment]
Yeah just tried it both ways with the crystal Rose and it didn't work at
all still overwriting Goldsmithing Moghancment.

**Comment by 59blargedy** [comment]
what else do you have in your mh that has energy? will help us figure it out, since im able to make it work with rose -- > fools gold.

**Comment by Kagekiki** [comment]
Works fine with everything but Marble plaque and Crystal rose. as far as
light goes. I am also afraid (haven't tried yet) that if i do multiple of
any light item it will overwrite fools gold.

**Comment by Kagekiki** [comment]
I haven't tried 2 dressers yet.
